partials/attributes.adoc

```diff
@@ -7,6 +7,8 @@
 :PRODUCT_LONG: IBM Business Automation Manager Open Editions
 :OPENSHIFT: OpenShift
 :VERSION: 9.0.0
+:VERSION901: 9.0.1
+:VERSION90X : 9.0.x
 :PRODUCT_VERSION: {VERSION}
 :VERSION_OLD: 8.0
```

Comment:
This may need to change to VERSION_8.0

Reply:
Fixed as VERSION_80 for consistency purpose, no dot in between